tests/level-db/test_leveldb_db_backend.py

```diff
@@ -0,0 +1,42 @@
+import os
+import pytest
+from evm.db.backends.memory import MemoryDB
+from evm.db import (
+    get_db_backend,
+)
+
+
+pytest.importorskip('leveldb') 
+memoryDb = MemoryDB()
```

Comment:
This looks like it should be a pytest fixture as it will currently share state across all the tests.

```python
@pytest.fixture
def memory_db():
    return MemoryDB()
```

Then in a test:

```python
def test_get(memory_db):
    ...
```

This way you get a fresh memorydb for every test case.